python/tests/test_node.py

```diff
@@ -76,7 +76,8 @@ def node_fn(x: int):
     options = srf.Options()
 
     # Set to 1 thread
-    options.topology.user_cpuset = "0-{}".format(pe_count)
+    # TODO(#38)
+    # options.topology.user_cpuset = "0-{}".format(pe_count)
```

Comment:
I cannot repo this locally and these changes are crucial to the test functioning correctly. This also has been passing in our CI. Do you have more info on what is causing this? I would prefer to keep these enabled if they are working in CI

Reply:
Will double check; it may have been fixed at some point with another patch.

Reply:
This looks like it was fixed somewhere else. Will close the other bug.